some adds

AndrewKetsko opened the pull request:
bg images added
prod images added
some styles added
.container changed to be adaptive (was responsive up to desktop)

AndrewKetsko committed:
some adds

bg images added
prod images added
some styles added
.container changed to be adaptive (was responsive up to desktop)

Nepalcem reviewed (changes requested):
Все ок, єдине є питання по ось цій влстивості - https://prnt.sc/0DdVtyTJ-ghl
.container {
  width: $mobile;
  
  У нас мобільна версія починається з 320px, якщо жорстко задати ширину 480px для мобілок, то може щось полізти , або з'явиться горизонтальний скролл при ширині екрану до 480px.

AndrewKetsko committed:
comment $mobile

Можливо і так, якщо хттсь десь буде жорстко задавати розміри. Закоментував цей рядок.

AndrewKetsko committed:
here it is

Ось воно, правильне виконання ширини контейнера )

Nepalcem commented:
ok
